### PR TITLE
WIP: Implement taxonomy term page 

### PR DESCRIPTION
**Pull request by HelenaEksler**
#133

### CODE REVIEW

**Comment by HelenaEksler** [comment]
@bitamar Should we show on taxonomy term page only books?

File: haskala/modules/custom/haskala_book/plugins/content_types/books_list/books_list.inc [hunk 1]
```diff
@@ -51,11 +56,42 @@ function haskala_book_books_list_content_type_render($subtype, $conf, $args, $co
...
  * @param $book_type
  *  Type of the book node. Could be regular or digital.
+ * @tid
+ *  Taxonomy term id.
  *
  * @return array
  *  Array with node objects.
  */
-function haskala_book_get_nodes_by_letter($letter, $limit, $node_type = 'book', $book_type = 'regular') {
+function haskala_book_get_nodes_by_letter($letter, $limit, $node_type = 'book', $book_type = 'regular', $tid = NULL) {
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/books_list/books_list.inc]: Let's have the new part as a separate function

File: haskala/modules/custom/haskala_book/plugins/content_types/books_list/books_list.inc [hunk 2]
```diff
@@ -51,11 +56,42 @@ function haskala_book_books_list_content_type_render($subtype, $conf, $args, $co
...
+
+    $query->fields('n',array('nid'))//SELECT the fields from node
+
+    ->condition('t.tid', $tid, '=')
+
+    ->condition('n.title', $letter . '%', 'LIKE')
+
+    ->orderBy('n.title', 'ASC');//ORDER BY title
+
+    if ($limit != 50) {
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/books_list/books_list.inc]: Add a variable for the limit, something like ```variable_get('haskala_index_items_limit', 50)```

**Comment by bitamar** [comment]
Closed in favour of #174

**Pull request closed without merging**